manufacturing-ai-suite/wind-turbine-anomaly-detection/grafana/dashboards/windturbine_dashboard.json

```diff
@@ -434,10 +588,10 @@
         "overrides": []
       },
       "gridPos": {
-        "h": 6,
+        "h": 5,
         "w": 4,
         "x": 20,
-        "y": 8
+        "y": 9

```

Comment:
Hey @sathyendranv ,
Let's add the new grafana dashboard snapshot in the docs, post that we can merge this.
It's